Makefile

```diff
@@ -86,6 +86,11 @@ config_local_test: $(DEPS) ## run all tests with workqueue_ex config
 	pip3 install ".[extreme_scale]"
 	PYTHONPATH=. pytest parsl -k "not cleannet" --config local --cov=parsl --cov-append --cov-report= --random-order
 
+.PHONY: site_test
+site_test:
+	pytest parsl -k "not cleannet" --config parsl/tests/site_tests/site_config_selector.py --cov=parsl --cov-append --cov-report= --random-order
+	pytest parsl/tests/site_tests/ --config local
```

Comment:
maybe should attempt to run the full set of `local` tests rather than the parsl/tests/site_test/ subdir unless there is a specific reason not to?

Reply:
We are running:
* full test suite against the site specific config
* Site specific tests that can't run on a single node that test provider behavior against the site's config.

We can run the full test suite with `--config local`, but I didn't think that added much more value, since we can just do that with `make config_local_test`. There's definitely some overlap here.